resources/dicts/events/death/general.json

```diff
@@ -3775,6 +3775,39 @@
             "lead_death": "{VERB/m_c/were/was} wounded by a fox"
         }
     },
+    {
+        "event_id": "gen_death_tree_anyleader",
+        "biome": [ "beach", "forest", "mountainous" ],
+        "camp": [
+            "any"
+        ],
+        "season": [
+            "any"
+        ],
+        "tags": [
```

Comment:
I think this event is only meant to take one life, so you can remove the "some_lives" tag.  By default, if no life tag is specified, an event will only remove one life from the leader.